docs/analytics-overview.md

```diff
@@ -0,0 +1,64 @@
+# Website Analytics Implementation Overview
+
+## Introduction
+
+This document provides a high-level overview of how the Improve workload and wellbeing for school staff service utilises
+Google Analytics 4 (GA4), Google Tag Manager (GTM), cookies, and a consent banner to track user interactions and comply
+with privacy regulations.
+
+## Google Analytics 4 (GA4)
+
+GA4 is the latest iteration of Google's analytics service, providing insights into website traffic, user behavior, and
+conversion metrics. We use GA4 to:
+
+- Monitor key metrics and performance indicators of our digital presence.
+- Understand user paths and interactions with our website.
+- Enhance marketing strategies and user experience based on data-driven insights.
+
+## Google Tag Manager (GTM)
+
+GTM is a tag management system that allows us to quickly and easily update tags and code snippets on our website, including:
+
+- Deployment and management of GA4 tracking codes.
+- Configuration of event tracking for user interactions without altering the source code.
+- Managing and testing marketing tags and related code snippets.
+
+## Cookies
+
+Cookies are small data files stored on the user's device, which we use to:
+
+- Remember user preferences and sessions.
+- Collect information that assists with personalizing the user experience.
+- Track user activity for analytics and advertising purposes in compliance with our privacy policy.
+
+## Consent Banner
+
+We respect user privacy and comply with legal requirements for data protection. As such, we implement a consent banner that:
+
+- Informs users about the use of cookies and tracking technologies on our website.
+- Allows users to give, or deny their consent for data collection.
+- Ensures that no tracking cookies are placed without explicit user consent, complying with GDPR, CCPA, and other data
+protection laws.
+
+## Consent Management and Data Collection Process
+
+When a user visits our website, the following occurs:
```

Comment:
We could also say somewhere in this section that before indicating their consent preference, `consentStatus` is set to `false`.